src/index.ts

```diff
@@ -1381,6 +1413,26 @@ class Remarkable implements RemarkableApi {
     return await this.bulkMove(hashes, "trash", refresh);
   }
 
+  /** permanent delete many hashes */
+  async bulkPurge(
+    hashes: readonly string[],
+    refresh: boolean = false,
+  ): Promise<[SimpleEntry[], string]> {
```

Comment:
the similar methods return the entries that were modified. Similarly it seems to make sense to return something relating to the entries that were actually deleted. These don't tend to return the root hash.

This API has evolved over a bit of time, so I'm open to an argument of why it helps to return something different, but baring that, I lean towards a consistent interface.

Reply:
I have no strong feelings on the return type. I was thinking about what could be useful data to return in this scenario and the only thing I could come up with was the root hash.

Reply:
One benefit of doing something like the the other returns is that it'll tell you which were actually removed, which would indicate that maybe things were already purged.